source/platforms/GitHub.ts

```diff
@@ -244,10 +245,10 @@ export class GitHub implements Platform {
       }
     } else {
       this.d(`Creating new comment`)
-      await this.createComment(newComment)
+      issue = await this.createComment(newComment)
     }
 
-    return true
+    return issue && issue.html_url
```

Comment:
this is a really smart way of avoiding accessing a property on `null`, although it wasn't very clear to me at first so I had to play with it to confirm that it does what I think it does 😄

Reply:
Yeah, this is definitely a JS-ism